tests/e2e/cluster_downgrade_test.go

```diff
@@ -243,3 +264,34 @@ func getMemberVersionByCurl(cfg *e2e.EtcdProcessClusterConfig, member e2e.EtcdPr
 	}
 	return result, nil
 }
+
+func generateSnapshot(t *testing.T, snapshotCount uint64, epc *e2e.EtcdProcessCluster) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	cc, err := e2e.NewEtcdctl(epc.Cfg.Client, epc.EndpointsGRPC())
+	assert.NoError(t, err)
+
+	var i uint64
+	t.Logf("Adding keys")
+	for i = 0; i < snapshotCount*3; i++ {
+		err := cc.Put(ctx, fmt.Sprintf("%d", i), "1", config.PutOptions{})
+		assert.NoError(t, err)
+	}
```

Comment:
Followup @geetasg. Could this function confirm that there is a snapshot file on disk present?

Reply:
updating.

Reply:
Awesome!